src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -1509,7 +1503,7 @@ def forward(
                 all_hidden_states = all_hidden_states + (hidden_states,)
 
             if self.gradient_checkpointing and self.training:
-                layer_outputs = checkpoint(
+                layer_outputs = self.gradient_checkpointing_func(
```

Comment:
here some models were using `torch.checkpoint` instead of `self.gradient_checkpointing_func` so I fixed it here